crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -26,28 +26,24 @@ use wgpu::{
 pub const INDEX_BUFFER_ASSET_INDEX: u64 = 0;
 pub const VERTEX_ATTRIBUTE_BUFFER_ID: u64 = 10;
 
-// TODO: allow values to be unloaded after been submitting to the GPU to conserve memory
-#[derive(Debug, TypeUuid, TypePath, Clone)]
-#[uuid = "8ecbac0f-f545-4473-ad43-e1f4243af51e"]
-pub struct Mesh {
-    primitive_topology: PrimitiveTopology,
-    /// `std::collections::BTreeMap` with all defined vertex attributes (Positions, Normals, ...)
-    /// for this mesh. Attribute ids to attribute values.
-    /// Uses a BTreeMap because, unlike HashMap, it has a defined iteration order,
-    /// which allows easy stable VertexBuffers (i.e. same buffer order)
-    attributes: BTreeMap<MeshVertexAttributeId, MeshAttributeData>,
-    indices: Option<Indices>,
-    morph_targets: Option<Handle<Image>>,
-    morph_target_names: Option<Vec<String>>,
-}
-
-/// Contains geometry in the form of a mesh.
+/// A 3D object made out of vertices representing triangles, lines, or points,
+/// with special "attribute" values for each vertex.
```

Comment:
I'm not sure I'd consider attributes to be special. They are an integral part of meshes.